src/main/java/fr/univartois/sonargo/coverage/CoverageSensor.java

```diff
@@ -52,6 +54,8 @@
 public class CoverageSensor implements Sensor {
 
     private static final Logger LOGGER = Loggers.get(CoverageSensor.class);
+    private Set<InputFile> inputFileWithCoverage = new HashSet<>();
+    private ProjectExplorer explorer;
 
```

Comment:
What about my previous proposal to make ProjectExplorer an utility class with static methods?

Reply:
We need to remove that ProjectExplorer field too.

Reply:
Indeed